electrophysiology/models.py

```diff
@@ -106,21 +118,26 @@ class SpikeSortedUnit(BaseExperimentalData):
     channel_group = models.IntegerField(null=True, blank=True,
                                         help_text="which shank this unit came from (an integer not a link)")
     trough_to_peak_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, computed from unfiltered mean waveform. TODO: which waveform?")
+                                 help_text="ms, computed from unfiltered mean spike waveform.")
     half_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, half width of negative peak in unfiltered waveform. TODO: which waveform?")
+                                 help_text="ms, half width of negative peak in unfiltered spike waveform.")
     trough_to_peak_amplitude = models.FloatField(null=True, blank=True,
-                                 help_text="µV, from filtered waveform. TODO: which waveform?")
+                                 help_text="µV, from filtered spike waveform.")
+								 # when you say which waveform, do you mean which channel? I guess the peak, but we can leave it unspecified...
     refractory_violation_rate = models.FloatField(null=True, blank=True,
-                                 help_text="fraction of spikes occurring < 2ms. TODO: is 2ms going to be hard-coded?")
+                                 help_text="fraction of spikes occurring < 2ms. ")
+								 # TODO: is 2ms going to be hard-coded?
+								 # i guess... what's the alternative?
```

Comment:
If the ArrayField is standardised (i.e. 50 bins wide, each of 1ms) then you can just do a
```
WHERE refractory_violation_rate[24] < 10 OR
refractory_violation_rate[25] < 10
```

for example. Or more likely, you'd write a function to calculate it for any width (default=2ms) and return these measures calculated on-the-fly in the REST API. Basically, the goaI is to duplicate as little information as possible - I can see definite advantages in storing a standardized CCG and mean peak channel waveform in the database, and then calculating all the stats from there on-the-fly...